src/Domain/Entity/Unfetched/UnfetchedProgramme.php

```diff
@@ -22,7 +23,8 @@ public function __construct()
             false,
             false,
             false,
-            0
+            0,
+            new Options()
```

Comment:
Should be UnfetchedOptions, as we don't know what the options are, rather than "we know them any they're empty"